File: backend/src/routes/health.ts
import { FastifyInstance } from "fastify";

export async function registerHealthRoute(app: FastifyInstance): Promise<void> {
  app.get("/health", async () => {
    return {
      service: "lumeris-backend",
      status: "ok",
      timestamp: new Date().toISOString()
    };
  });
}

File: backend/app/routers/health.py
from datetime import datetime, timezone

from fastapi import APIRouter

from app.schemas import HealthResponse

router = APIRouter(tags=["health"])


@router.get("/health", response_model=HealthResponse)
def health() -> HealthResponse:
    return HealthResponse(
        service="lumeris-backend",
        status="ok",
        timestamp=datetime.now(timezone.utc).isoformat(),
    )
